.github/workflows/release.yml

```diff
@@ -72,4 +72,27 @@ jobs:
         run: npx -p jsii-release jsii-release-pypi
         env:
           TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
\ No newline at end of file
+          TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
+          
+  release_homebrew:
+    name: Release to Homebrew
+    # The branch or tag ref that triggered the workflow run.
+    if: startsWith(github.event.head_commit.message, 'Bump version')
+    needs: release_npm
+    runs-on: ubuntu-latest
+    steps:
+      # extract version number from package.json
+      - uses: actions/checkout@v2
+      - name: version
+        id: get_version
+        run: |
+          version=$(node -p "require('./package.json').version")
+          echo "::set-output name=version::${version}"
+      # A PR will be sent to github.com/Homebrew/homebrew-core to update this formula:
+      - uses: mislav/bump-homebrew-formula-action@v1
+        with:
+          formula-name: cdktf
+          download-url: https://registry.npmjs.org/cdktf-cli/-/cdktf-cli-${{ steps.get_version.outputs.version }}.tgz
+          commit-message: cdktf ${{ steps.get_version.outputs.version }}
+        env:
+          COMMITTER_TOKEN: ${{ secrets.HOMEBREW_COMMITTER_TOKEN }}
```

Comment:
I will generate a GitHub token with "public" scope for this one.

Reply:
Has this token been set already?

Reply:
I have saved the token for hashicorp-ci GitHub user.